src/layouts/Footer/Footer.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import style from './Footer.module.scss';
```

Comment:
i think it is better to to name classes, cos u import not only one

Reply:
It may be treated as in "style object". But TBH I agree with Dima. let's keep it plural just for the sake of "not confusing with style property or class reserved word". So either styles or classes

Reply:
Understood. Corrected to classes